lib/cfn-nag/custom_rules/IamPolicyPassRoleWildcardResourceRule.rb

```diff
@@ -13,7 +13,7 @@ def rule_type
   end
 
   def rule_id
-    'F39'
+    'F41'
```

Comment:
is there a good reason to change this id?  this change could screw up a "compliance" report

Reply:
This was because I started the creation of the `AmplifyAppAccessTokenRule` rule way back before this iam policy was implemented. I went through and changed the wrong duplicate rule_id. This one should stay unchanged and the new rule should take the currently unused rule_id `F41`.